mds/apis/agency_api/v0_x/vehicles.py

```diff
@@ -187,6 +192,12 @@ def create(self, validated_data):
                 "trip_id": validated_data.get("trip_id"),
             },
         )
+        utils.upsert_event_records([event_record], "push", on_conflict_update=True)
+
+        # XXX How to get the created or updated object?
+        return models.EventRecord.objects.get(
+            device=device, timestamp=validated_data["timestamp"]
+        )
```

Comment:
no choice but to query again - or return nothing if DRF serialiser would take it.